requirements.txt

```diff
@@ -8,3 +8,4 @@ pillow<=11.2.1,>=10.4.0
 requests==2.32.3
 bokeh<=3.7.2,>=3.1.1
 protobuf<=6.30.2,>=5.29.4
+swig<=4.3.1,>=4.2.1
```

Comment:
This requirement should be in `requirements_dev.txt`, not this file. This file should only have runtime requirements, not requirements to build the software.